RELEASE.md

```diff
@@ -1,25 +1,32 @@
 # Manual release process
 
-1. Create a new branch for the release `git checkout -b release-2023.9.1`
-2. Clean the branch `git clean -fxdq`
-3. Increment version in `package.json`
-4. Build the package locally:
+1. Open a new release issue in the repository, following the [release issue template](../.github/ISSUE_TEMPLATE/release.md).
+1. Create a new branch for the release `git checkout -b release-2024.9.1`
+1. Clean the branch `git clean -fxdq`
+1. Increment the version in `package.json`
+1. Build the package locally:
```

Comment:
Why repeat the same info from the issue template?

Reply:
People sometimes look for the release.md file if curious vs. an issue template, so it would not hurt to keep this

Reply:
My concern is more about the repeated content, which then has to be updated in two places instead of one if these two documents are not going to get out of sync